cardano-api/src/Cardano/Api/LedgerState/Serialisation.hs

```diff
@@ -0,0 +1,46 @@
+{-# LANGUAGE ExplicitNamespaces #-}
```

Comment:
Does this really deserve its own module?

Reply:
I can move it to `Cardano.Api.LedgerState`. In the original version the serializer was much more complex, but it has since been simplified, so there's less reason for a separate module.

Reply:
@Jimbo4350 Done.